### PR TITLE
Rename v4-pci and v6-pci args

### PR DESCRIPTION
**Pull request by dpino**
Rename v4-pci and v6-pci arguments to v4 and v6.

While https://github.com/Igalia/snabb/pull/298 doesn't land yet, I've renamed the v4-pci and v6-pci arguments to v4 and v6. Documentation has been updated and the short options for these arguments ('n' and 'm' respectively) has been removed as they were never used.

### CODE REVIEW

**Comment by aperezdc** [comment]
LGTM. Once this is merged I can rebase #298.

File: src/program/lwaftr/run/run.lua [hunk 1]
```diff
@@ -68,24 +68,26 @@ function parse_args(args)
...
+      if not v6 then
+         fatal("Argument '--v6' was not set")
       end
-      if not nic_exists(v6_pci) then
-         fatal(("Couldn't locate NIC with PCI address '%s'"):format(v6_pci))
+      if not nic_exists(v6) then
+         fatal(("Couldn't locate NIC with PCI address '%s'"):format(v6))
       end
    end
+   handlers["v6-pci"] = handlers.v6
```
Comment (wingo) [inline src/program/lwaftr/run/run.lua]: LGTM.  Would you mind making --v4-pci and --v6-pci print a warning before dispatching to the v4/v6 handler?  That will get our users migrating to the new argument name.  Thank you!

**Comment by dpino** [comment]
OK, done. The warning message looks like this:

```
$ sudo ./snabb lwaftr run --conf lwaftr.conf --v4-pci 82:00.0 --v6-pci 82:00.1
WARNING: Deprecated argument '--v4-pci'. Use '--v4' instead.
WARNING: Deprecated argument '--v6-pci'. Use '--v6' instead.
```

**Comment by wingo** [comment]
Looks great to me!

**Pull request merged**